togglwrapper/api.py

```diff
@@ -206,13 +206,19 @@ def get_clients(self, workspace_id):
         """ Gets the Clients for the Workspace with the given ID. """
         return super(Workspaces, self).get(workspace_id, '/clients')
 
-    def get_projects(self, workspace_id):
+    def get_projects(self, workspace_id, active=None, actual_hours=None, only_templates=None):
         """ Gets the Projects for the Workspace with the given ID. """
-        return super(Workspaces, self).get(workspace_id, '/projects')
+        params = {}
+        params.update(self.param_bool_both('active', active))
+        params.update(self.param_bool('actual_hours', actual_hours))
+        params.update(self.param_bool('only_templates', only_templates))
```

Comment:
This is making me realize it would be helpful to have a rudimentary type checking system in place, thank you. I'll see about implementing one.